src/subcommand/wallet/balance.rs

```diff
@@ -0,0 +1,20 @@
+use super::*;
+
+#[derive(Debug, Parser)]
+pub(crate) struct Balance {}
+
+impl Balance {
+  pub(crate) fn run(self, options: Options) -> Result {
+    println!(
+      "{}",
+      options
+        .bitcoin_rpc_client_for_wallet_command("ord wallet balance")?
+        .get_balances()?
+        .mine
+        .trusted
+        .to_sat()
+    );
+
+    Ok(())
+  }
+}
```

Comment:
Indentation is wrong, but this should be something like:

```suggestion
  pub(crate) fn run(options: Options) -> Result {
    println!(
      "{}",
      options
        .bitcoin_rpc_client_for_wallet_command("ord wallet balance")?
        .get_balances()?
        .mine
        .trusted
        .to_sat()
    );

    Ok(())
  }
```

Since we don't need a struct. I also think this can use `Options::bitcoin_rpc_client`, since this command is pretty safe, and we don't need to enforce not running it on mainnet or on wallets with large balances.

Reply:
:+1: